examples/tools/internal_ambiguities.rs

```diff
@@ -0,0 +1,11 @@
+//! Reports any Bevy-internal system-order ambiguities
+//! This is primarily useful for engine development
+
+use bevy::prelude::*;
+
+fn main() {
+    App::new()
+        .add_plugins(DefaultPlugins)
+        .insert_resource(ExecutionOrderAmbiguities::Forbid)
```

Comment:
Should this be `WarnInternal` instead of `Forbid`?

Reply:
```suggestion
        .insert_resource(ExecutionOrderAmbiguities::WarnInternal)
```